### PR TITLE
Fix header img margin quirk

### PR DESCRIPTION
**Pull request by skeuomorf**
refs #368 

This is a workaround (super duper hack) to fix the issue in #368

### CODE REVIEW

**Comment by sigmavirus24** [comment]
Oh let's please not use negative anything if we can find a better way. That is just a recipe for disaster. I'll take a look at this right now to see if I can narrow down which element is doing this.

File: pycon/static/less/site.less [hunk 1]
```diff
@@ -391,6 +391,7 @@ aside {
         -moz-transform: translate(-50%,0);
         -ms-transform: translate(-50%,0);
```
Comment (sigmavirus24) [inline pycon/static/less/site.less]: So these transforms are exactly what's causing this. If you remove them and then decrease `margin-left` to `5%` the site looks the same and the problem disappears. Please remove the negative `margin-right`.

To be clear, this works on Chrome and Firefox. I haven't tested Safari, IE, or any of the mobile browsers.

**Comment by skeuomorf** [comment]
@sigmavirus24 Totally agree, I was in a hurry when I did PR, hence "super duper hack" :)

File: pycon/static/less/site.less (hunk 1 above)
Comment (skeuomorf) [inline pycon/static/less/site.less]: It appears that when I remove the `transform` attributes and both `margin-left` and `margin-right` it works perfectly, tested on Chrome and Firefox on Linux. Will update.

**Comment by sigmavirus24** [comment]
Without a margin-left the image didn't show up properly for me when I was toying with the live site in developer tools, but this sounds roughly correct anyway. Can anyone else verify this works for them? (Perhaps on other browsers, e.g., Safari, IE)

**Comment by rebecca-caktus** [comment]
:+1: from me and :+1: from @rayt86. I'll merge this and send it to the staging site, then double-check it on mobile browsers & IE.

**Comment by rebecca-caktus** [comment]
Many, many thanks @skeuomorf and @sigmavirus24!

**Comment by sigmavirus24** [comment]
Thanks @rebecca-caktus !

**Comment by skeuomorf** [comment]
Thanks @rebecca-caktus :)

**Comment by rebecca-caktus** [comment]
Deployed to production. Thanks again!

**Pull request merged**